### DISCUSSION TITLE
Sync help with readme

### DISCUSSION THREAD

**Discussion by c-git** [rust-practice/cargo-leet#56] [Ideas]
Would be nice if we could find an "easy" way to keep the first paragraph of the readme in sync with what shows at the command line in the help

**Comment by c-git** [comment]
In addition would be nice to sync the readme with the docs on docs.rs. Projects to look into in that regard.
- https://github.com/livioribeiro/cargo-readme
- https://scrabsha.github.io/abcr-issue-0/

**Reply by scrabsha** [reply]
You may look at https://github.com/msrd0/cargo-doc2readme as well (IIRC it uses `syn` to parse the `lib.rs` file or its expanded version)

**Reply by c-git** [reply]
Thanks
